operate/client/src/modules/utils/incidents.ts

```diff
@@ -34,6 +34,10 @@ const ERROR_TYPE_NAMES: Record<IncidentErrorType, string> = {
   RESOURCE_NOT_FOUND: 'Resource not found.',
 };
 
+const availableErrorTypes = Object.keys(
+  ERROR_TYPE_NAMES,
+) as IncidentErrorType[];
```

Comment:
The suggestion below doesn't work?

```suggestion
const availableErrorTypes = Object.keys(
  ERROR_TYPE_NAMES,
) satisfies IncidentErrorType[];
```